data_hub_api/docmaps/v2/sql/docmaps_index.sql

```diff
@@ -302,11 +302,50 @@ t_rp_meca_path_update AS (
   WHERE rn=1
 ),
 
+t_manual_preprint_match_for_published_date AS (
+  SELECT
+    * EXCEPT(rn),
+  FROM
+  (
+    SELECT 
+      *,
+      ROW_NUMBER() OVER(PARTITION BY long_manuscript_identifier ORDER BY imported_timestamp DESC) AS rn
+    FROM `elife-data-pipeline.prod.unmatched_manuscripts`
+    WHERE preprint_published_at_date IS NOT NULL 
+      AND preprint_published_at_date != ''
+  )
+  WHERE rn = 1
+),
+
+t_europepmc_preprint_publication_date AS (
+  SELECT
+    response.doi,
+    response.firstPublicationDate,
+    CASE
+      WHEN url_list.url LIKE '%10.1101/%v%'
```

Comment:
The field is called `firstPublicationDate`. Somehow I thought it would be the first publication date for the given DOI. Which for bioRxiv would mean the first version. Have you checked the behaviour?

Reply:
For bioRxiv, don't we get the publication dates from the bioRxiv API?